pkgs/tools/misc/screen/default.nix

```diff
@@ -16,7 +16,12 @@ stdenv.mkDerivation rec {
     "--enable-colors256"
   ];
 
-  patches = lib.optional stdenv.hostPlatform.isMusl
+  patches = [
+    (fetchpatch {
```

Comment:
nit, but I'd add something to the effect of

```
Patch sourced from Debians package, originating here: https://lists.gnu.org/archive/html/screen-devel/2021-02/msg00010.html
```

as a comment above the patch, as well as setting `name = "CVE-2021-26937.patch";` (this seems to be the norm for CVE patches).